### PR TITLE
{tools}[foss/2020b] neptune-client v0.10.5 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/831fe4a7a9342145d43ec969e41e6131 for a full test report.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+use_pip = True
+
+exts_list = [
+    ('monotonic', '1.6', {
+        'checksums': ['3a55207bcfed53ddd5c5bae174524062935efed17792e9de2ad0205ce9ad63f7'],
+    }),
+    ('msgpack', '1.0.2', {
+        'checksums': ['fae04496f5bc150eefad4e9571d1a76c55d021325dcd484ce45065ebbdd00984'],
+    }),
+    ('pyrsistent', '0.18.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb:30]: 0.17.3 in Python 3.8.6 - is the updated version necessary?

File: easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    ('swagger-spec-validator', '2.7.3', {
+        'checksums': ['f4f23ee4dbd52bfcde90b1144dde22304add6260e9f29252e9fd7814c9b8fd16'],
+    }),
+    ('jsonref', '0.2', {
+        'checksums': ['f3c45b121cf6257eafabdc3a8008763aed1cd7da06dbabc59a9e4d2a5e4e6697'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('simplejson', '3.17.4', {
```
Comment (branfosj) [inline easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb:42]: 3.17.2 in Python 3.8.6 - is the updated version necessary?

File: easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('monotonic', '1.6', {
+        'checksums': ['3a55207bcfed53ddd5c5bae174524062935efed17792e9de2ad0205ce9ad63f7'],
+    }),
+    ('msgpack', '1.0.2', {
```
Comment (branfosj) [inline easybuild/easyconfigs/n/neptune-client/neptune-client-0.10.5-foss-2020b.eb:27]: 1.0.0 in Python 3.8.6 - is the updated version necessary?

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
@branfosj I totally forgot to check the changes in base Python in `2020b`, thanks!

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c7bf830ca6459049115cd21d874187a5 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7898c209007c3304882b1afaac7a893a for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @lexming!

**Pull request merged**